### PR TITLE
updated for card #2266

### PR DESCRIPTION
**Pull request by Jacqueline-Yarbrough**
so the ABL reflects the redundant "check your understanding" text

### CODE REVIEW

File: approved-book-list.json [hunk 1]
```diff
@@ -1714,6 +1729,11 @@
       "content_version": "1.3.5",
       "min_code_version": "99999999.999999"
     },
+    {
+      "collection_id": "col10280",
+      "content_version": "1.5.2",
+      "min_code_version": "99999999.999999"
+    },
```
Comment (rnathuji) [inline approved-book-list.json:1736]: Just FYI - You don't necessarily have to add this while the book is only for building in CORGI. We need one entry, but it doesn't really matter that it matches the version you build in CORGI.

**Review by rnathuji — COMMENTED** [review]
Check my 👀 , but I think the CI is failing because all of the collection versions except for `col10280` are duplicates 🤷‍♂️

**Comment by m1yag1** [comment]
@Jacqueline-Yarbrough your probably going to need to update this branch with latest code because you have a conflict. Additionally, @rnathuji had some comments. I'm mainly commenting because I've seen this in the queue but hasn't moved in a while.

File: approved-book-list.json (hunk 1 above)
Comment (Jacqueline-Yarbrough) [reply approved-book-list.json:1736]: hello i'm so sorry i've neglected this for so long. would removing this entry resolve it?

Comment (rnathuji) [reply approved-book-list.json:1736]: Actually, I wonder if this PR can just be closed 🤷‍♂️ ? I believe all of the entries are either duplicates or unnecessary.

Comment (Jacqueline-Yarbrough) [reply approved-book-list.json:1736]: yes let's go ahead and do that. can you do that from your end or do i need to submit it?

Comment (rnathuji) [reply approved-book-list.json:1736]: I can close it out.

Comment (Jacqueline-Yarbrough) [reply approved-book-list.json:1736]: thank you!

**Pull request closed without merging**